functionary/ui/scss/custom.scss

```diff
@@ -32,9 +35,10 @@ $dark: #343E40;
   ------------------*/
 $enable-rfs: false; // Disable Responsive Font Sizes
 
-$navbar-dark-color: $secondary;
-$navbar-dark-hover-color: #D3DAE4; // Slight deviation from default for compliance
+$navbar-dark-color: rgba($gray-200, .75);
+$navbar-dark-hover-color: rgba($white, .75);
```

Comment:
This hover color seems pretty indistinguishable from the `navbar-dark-color`.  Two options that look okay to me just from messing around:

* Go darker, with .35 opacity.
* Go full $white.  Adding "fw-semibold" in addition to "active" could help slightly differentiate the currently active from the hover over if we go this route.